### PR TITLE
WL-485: Basket bulk purchase view updated

### PR DESCRIPTION
**Pull request by malikshahzad228**
@mattdrayer @saleem-latif @asadiqbal08 

Item Seat price and view updated.

[WL-485](https://openedx.atlassian.net/browse/WL-485)

MIT Theme PR: https://github.com/edx/edx-themes/pull/41

### CODE REVIEW

**Comment by malikshahzad228** [comment]
Screen Shots:

wl-485 bulk full screen

wl-485 bulk ipad

wl-485 bulk phone screen

wl-485 single seat full screen

wl-485 single seat ipad

wl-485 single seat mobile

**Comment by mattdrayer** [comment]
@asadiqbal08 if you can please take a look at this PR, that would be great -- thanks!

File: ecommerce/static/js/pages/basket_page.js [hunk 1]
```diff
@@ -93,6 +93,26 @@ define([
                 checkoutPayment(data);
             });
 
+            $('.spinner .btn:first-of-type').on('click', function() {
```
Comment (asadiqbal08) [inline ecommerce/static/js/pages/basket_page.js:97]: please add some comments that describe the functionality ?

File: ecommerce/static/js/pages/basket_page.js [hunk 2]
```diff
@@ -93,6 +93,26 @@ define([
...
+                var btn = $(this);
+                var input = btn.closest('.spinner').find('input');
+                if (input.attr('max') === undefined || parseInt(input.val()) < parseInt(input.attr('max'))) {
+                    input.val(parseInt(input.val()) + 1);
+                } else {
+                    btn.next('disabled', true);
+                }
+            });
+
+            $('.spinner .btn:last-of-type').on('click', function() {
```
Comment (asadiqbal08) [inline ecommerce/static/js/pages/basket_page.js:109]: please add some comments that describe the functionality ? what is going here ?

File: ecommerce/templates/oscar/basket/partials/basket_content.html [hunk 3]
```diff
@@ -22,37 +22,50 @@
...
-                                            {{benefit_value}} off
-                                        {% endblocktrans %}
-                                    </div>
-                                    <div class="old-price">
-                                        {{ line_data.line.line_price_incl_tax|currency:line_data.line.price_currency }}
+                                <div class="col-md-3 col-xs-12 form-inline">
+                                    <label class="course-price-label text-muted">{% trans 'Quantity' %}</label>
+                                    <div class="checkout-quantity form-group">
+                                        <div class="input-group spinner  {% if form.errors %}error{% endif %}">
+                                            {% render_field form.quantity class+="quantity form-control" min=min_seat_quantity %}
```
Comment (asadiqbal08) [inline ecommerce/templates/oscar/basket/partials/basket_content.html:42]: It is the quantity is read-only field  ?

Comment (malikshahzad228) [inline ecommerce/templates/oscar/basket/partials/basket_content.html:42]: No, its not readonly. User can update field by keyboard as well.

File: ecommerce/static/js/pages/basket_page.js (hunk 1 above)
Comment (malikshahzad228) [inline ecommerce/static/js/pages/basket_page.js:97]: @asadiqbal08 Comments added in code.

File: ecommerce/templates/oscar/basket/partials/basket_content.html (hunk 3 above)
Comment (asadiqbal08) [inline ecommerce/templates/oscar/basket/partials/basket_content.html:42]: so what will happened, If user enters the value greater then Max limit ?

File: ecommerce/static/js/pages/basket_page.js [hunk 4]
```diff
@@ -93,6 +93,30 @@ define([
                 checkoutPayment(data);
             });
 
+            // Increment the quantity field until max
+            $('.spinner .btn:first-of-type').on('click', function() {
+                var btn = $(this);
+                var input = btn.closest('.spinner').find('input');
+                // Stop if max attribute is defined and value reached to given max value
```
Comment (asadiqbal08) [inline ecommerce/static/js/pages/basket_page.js]: nit: 'value is reached' same below LN112

File: ecommerce/templates/oscar/basket/partials/basket_content.html (hunk 3 above)
Comment (malikshahzad228) [inline ecommerce/templates/oscar/basket/partials/basket_content.html:42]: HTML form validation will prevent from submitting the form, and HTML form validation message will be shown.

**Comment by saleem-latif** [comment]
LGTM :+1:

**Comment by asadiqbal08** [comment]
👍

**Pull request merged**